src/eradiate/test_tools/test_cases/atmospheres.py

```diff
@@ -0,0 +1,140 @@
+import numpy as np
+
+from eradiate.experiments import AtmosphereExperiment
+from eradiate.units import unit_registry as ureg
```

Comment:
Prefer relative imports when you are in the `eradiate` package:
```suggestion
from ...experiments import AtmosphereExperiment
from ...units import unit_registry as ureg
```